app/routes.js

```diff
@@ -152,22 +168,33 @@ router.post('/finish', function(req, res){
     //TODO submit the form into a database and send email via Notify
     var _userinfo = sess.user_info
 
-    //notify the user
+    // LOG _userinfo
+    console.log(_userinfo)
+
+    // SAVE TO DATABASE
+    db.collection('booking').save(_userinfo, (err, result) => {
+        if (err) return console.log(err)
+        console.log('saved to database')
+    })
+
+    // NOTIFY - USER
     notifyClient.sendEmail("22098707-7d16-453a-8e02-baecc466c2d5", _userinfo.email, {
-      'service': _userinfo.service_name,
+      'service_name': _userinfo.service_name,
       'service_manager': _userinfo.name,
       'assessment_date': _userinfo.assessment_date,
-      'stage': _userinfo.assessment_stage,
-      'assessment_time': "AM"
+      'service_stage': _userinfo.assessment_stage,
+      'assessment_time': _userinfo.assessment_time,
+      'briefing_deadline': _userinfo.assessment_date
     })
 
-    //notify the team
+    // NOTIFY - SERVICE ASSURANCE TEAM
     notifyClient.sendEmail("22098707-7d16-453a-8e02-baecc466c2d5", team_email, {
```

Comment:
It's the email template ID so unique to the email we're sending.  Have checked with the Notify team and there's no need to treat as a secret.